sky/shell/BUILD.gn

```diff
@@ -389,8 +388,7 @@ if (is_android) {
     ]
   }
 
-  sky_ios_sdk("shell") {
-    sdk_name = "Flutter"
+  group("shell") {
   }
```

Comment:
We probably need to make this depend on something in order to trigger Flutter.framework to build on iOS